<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveProjectDialog</class>
 <widget class="QDialog" name="SaveProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>161</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="media.qrc">
    <normaloff>:/media/document_save_all.png</normaloff>:/media/document_save_all.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="projectNameLabel">
       <property name="text">
        <string>Project's Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="projectDirLabel">
       <property name="text">
        <string>Project's directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectDirEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="projectDirBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset resource="media.qrc">
         <normaloff>:/media/document_open.png</normaloff>:/media/document_open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections/>
</ui>
